src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__staff_renewal_feed.sql

```diff
@@ -0,0 +1,98 @@
+with
+    step_scales as (
+        select
+            sr.employee_number,
+            pss.salary_rule,
+            pss.scale_cy_salary,
+            pss.scale_ny_salary,
+            pss.scale_step,
+            pss.school_level,
+        from {{ ref("base_people__staff_roster") }} as sr
+        left join
+            {{ source("people", "src_people__salary_scale") }} as pss
+            on sr.job_title = pss.job_title
+            and sr.business_unit_assigned_name = pss.region
+            and sr.base_remuneration_annual_rate_amount_amount_value + 150
+            >= pss.scale_cy_salary
+            and sr.base_remuneration_annual_rate_amount_amount_value + 150
+            < pss.scale_ny_salary
+            and pss.school_level is null
+        where sr.assignment_status = 'Active' and pss.school_level is null

```

Comment:
you can move `pss.school_level is null` to the join and make it an `inner join`

Reply:
I don't want this to be an inner join because src_people__salary_scale does not have an exhaustive list of jobs on it